binding/rust/README.md

```diff
@@ -0,0 +1,91 @@
+# Porcupine Wake Word Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Porcupine is a highly-accurate and lightweight wake word engine. It enables building always-listening voice-enabled
+applications. It is
+
+- using deep neural networks trained in real-world environments.
+- compact and computationally-efficient. It is perfect for IoT.
+- cross-platform. Raspberry Pi, BeagleBone, Android, iOS, Linux (x86_64), macOS (x86_64), Windows (x86_64), and web
+browsers are supported. Additionally, enterprise customers have access to ARM Cortex-M SDK.
+- scalable. It can detect multiple always-listening voice commands with no added runtime footprint.
+- self-service. Developers can train custom wake word models using [Picovoice Console](https://picovoice.ai/console/).
+
+## Compatibility
+
+- Rust 1.54+
+- Runs on Linux (x86_64), macOS (x86_64), Windows (x86_64), Raspberry Pi, NVIDIA Jetson (Nano), and BeagleBone
+
+## Installation
+First you will need [Rust and Cargo](https://rustup.rs/) installed on your system.
+
+To add the porcupine library into your app, add `pv_porcupine` to your apps `Cargo.toml` manifest:
+```toml
+[dependencies]
+pv_porcupine = "1.9.1"
+```
+
+If you prefer to clone the repo and use it locally, first run `copy.sh` (**NOTE:** on Windows, Git Bash or another bash shell is required, or you will have to manually copy the libs into the project.). Then you can reference the local binding location:
+```toml
+[dependencies]
+pv_porcupine = { path = "/path/to/rust/binding" }
+```
+
+## Usage
+
+To create an instance of the engine you first create a PorcupineBuilder instance with the configuration parameters for the wake word engine and then make a call to `.init()`.
+
+```rust
+use porcupine::{BuiltinKeywords, PorcupineBuilder};
+
+let porcupine: Porcupine = PorcupineBuilder::new_with_keywords(&[BuiltinKeywords::Porcupine]).init().expect("Unable to create Porcupine");
+```
+In the above example, we've initialzed the engine to detect the built-in wake word "Porcupine". Built-in keywords are contained in the package with the `BuiltinKeywords` enum type.
+
+Porcupine can detect multiple keywords concurrently:
+```rust
+let porcupine: Porcupine = PorcupineBuilder::new_with_keywords(&[BuiltinKeywords::Porcupine, BuiltinKeywords::Blueberry, BuiltinKeywords::Bumblebee])
+    .init().expect("Unable to create Porcupine");
+```
+
+To detect non-default keywords, use `PorupineBuilder`'s `new_with_keyword_paths` method instead:
```

Comment:
*custom keywords

Reply:
Also, mention here that they should pass in a .ppn file just to be a bit more clear